WordPress/Tests/Theme/NonPrintableCheckUnitTest.css

```diff
@@ -0,0 +1,8 @@
+/* 
```

Comment:
For ease of understanding what you're testing for on each line, it would be great if you would add some indication of the character "hidden" in each line. Something like `Bad - ASCII 07`.
This goes for all three test files.

Also, please mind punctuation in the test file. The _Good/Bad_ comments should generally start with a capital and end with a `.` period.